debian/libcephsqlite-dev.install

```diff
@@ -0,0 +1 @@
+usr/include/libcephsqlite.h
```

Comment:
i think we should install `usr/lib/libcephsqlite.so` here. so developer can link against the latest shared library instead of using the one shipped by `libcephsqlite`.

Reply:
Wouldn't that cause the -dev package to conflict with the non-dev one?

Reply:
no. the general practice is to 

- let libfoo-dev install .so
- let libfoo install .so.1